### PR TITLE
Doc(eos_cli_config_gen): Add ip_ssh_client_source_interfaces to README

### LINKED ISSUE #1151: Add support for SSH client-interface
### Enhancement summary

This command is currently not supported to explicitly set the ssh client source:

```
CL-1-09:04:14(config-mgmt-ssh)#ip ssh client source-interface ?
  Ethernet      Ethernet Sub Interface
  Loopback      Hardware interface used for looping packets
  Management    Management interface
  Port-Channel  Port-Channel Sub Interface
  Tunnel        Tunnel interface
  Vlan          Logical interface into a VLAN
```

Doc:
https://avd.sh/en/releases-v3.x.x/roles/eos_cli_config_gen/index.html#management-ssh

### Which component of AVD is impacted

eos_cli_config_gen

### Use case example

Customer has this running already in devices not managed by AVD.

### Describe the solution you would like

AVD support for this command.

### Describe alternatives you have considered

_No response_

### Additional context

_No response_

### Contributing Guide

- [X] I agree to follow this project's Code of Conduct

### PR DESCRIPTION
**Pull request by tgodaA**
## Change Summary
Add ip_ssh_client_source_interfaces description to eos_cli_config_gen/README.md

## Related Issue(s)
Fixes #1151 
Related PR #1212

## Component(s) name

`arista.avd.eos_cli_config_gen`

### User Checklist

- N/A

### Repository Checklist

- [x] My code has been rebased from devel before I start
- [x] I have read the [**CONTRIBUTING**](https://avd.sh/en/latest/docs/contribution/overview.html) document.
- [x] My change requires a change to the documentation and documentation have been updated accordingly.
- [ ] I have updated [molecule CI](https://github.com/aristanetworks/ansible-avd/tree/devel/ansible_collections/arista/avd/molecule) testing accordingly. (check the box if not applicable)

### CODE REVIEW

**Review by carlbuchmann — APPROVED** [review]
LGTM

**Review by titom73 — APPROVED** [review]
LGTM!

File: ansible_collections/arista/avd/roles/eos_cli_config_gen/README.md [hunk 1]
```diff
@@ -1415,6 +1416,16 @@ management_ssh:
       enable: < true | false >
 ```
 
+#### IP SSH Client Source Interfaces
+
+```yaml
+ip_ssh_client_source_interfaces:
+    - name: <interface_name_1>
+      vrf: <vrf_name_1 | optional set to default>
```
Comment (ClausHolbechArista) [inline ansible_collections/arista/avd/roles/eos_cli_config_gen/README.md]: please use same wording as we do elsewhere. So `vrf: < vrf_name_1 | default -> "default" >`

**Review by ClausHolbechArista — APPROVED** [review]
LGTM

**Pull request merged**